kythe/java/com/google/devtools/kythe/extractors/java/standalone/AbstractJavacWrapper.java

```diff
@@ -127,6 +116,28 @@ public void process(String[] args) {
     }
   }
 
+  private static void outputIndexInfo(CompilationDescription indexInfo) throws IOException {
+    String outputFile = System.getenv("KYTHE_OUTPUT_FILE");
```

Comment:
Same as above.  We wouldn't need separate env vars if the extractor inferred the output format from the file extension.  I think makes more sense since the `.kindex` file is being deprecated anyway.

Reply:
Works for me. I was worried that somewhere we might have some kindex files without an extension.